express-post-json/index.js

```diff
@@ -0,0 +1,24 @@
+const grades = [];
+let nextId = 1;
+
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
+
+app.post('/api/grades', (req, res) => {
+  res.status(201);
+  req.body['id:'] = `${nextId}`;
+  grades.push(req.body);
+  nextId++;
+  res.json(grades);
```

Comment:
The server should respond with the added grade and not the whole list.

Reply:
Oh, missed this comment. Will fix.